content/docs/scaffolder/writing-templates/index.md

```diff
@@ -14,6 +14,16 @@ A Scaffolder template is then run on demand by the users of Backstage to execute
 
 You can find a step by step guide to adding templates in Roadie [here](/docs/getting-started/scaffolding-components/)
 
+<div class="docs-cta">
+  <h4 class="docs-cta__title">Want to see the Scaffolder in action?</h4>
+  <p class="docs-cta__paragraph">
+    Discover more self-service possibilities by the hand of a Backstage expert. 
```

Comment:
The English is slightly off here. Something like "in the hands of" would read a little more native.

You might be able to do without this sentence entirely though. Perhaps "See the Scaffolder in action" is enough?

Reply:
You're right, I was trying to force the "Backstage expert" part. I do want to have a little paragraph here so they don't think they'll be going to some kind of playground

Reply:
I ended up removing it because the page they get to already has this copy so let's bet for shorter as you suggested.